components/ruby/spec/restful_client_v1_spec.rb

```diff
@@ -16,6 +17,7 @@
         config.license_server_url_check_in_file = true
         config.chef_product_name = "inspec"
         config.chef_entitlement_id = "3ff52c37-e41f-4f6c-ad4d-365192205968"
+        config.cache_enabled = false
```

Comment:
Could this be a global setting or simply a before block to set it before every example

Reply:
This could be refactored to disable the cache for all the examples, except for the ones where it is required enabling. I made a fix for the tests which were breaking without much effort.

Reply:
I would prefer refactoring the tests as part of a different PR.